15/index.php

```diff
@@ -10,21 +10,24 @@
 // Required
 head([
   'class' => 'keyman11',
-  'title' => 'Keyman 15 is now in Beta!',
+  'title' => 'Keyman 15 is now available!',
   'css' => ['template.css', 'dev.css', 'app-store-links.css', 'prism.css'],
   'js' => ['prism.js'],
   'showMenu' => true,
   'banner' => [
-  'title' => 'Keyman 15.0 Beta',
+  'title' => 'Keyman 15.0 is now available!',
     'image' => 'screenshots/14/typewriter.jpg',
     'background' => 'black'
   ]
 ]);
 ?>
 
-<h2>Announcing the release of Keyman 15.0 Beta</h2>
-<p class="red">2 March 2022</p>
-<p>We are pleased to announce that Keyman 15.0 is now in beta!</p>
+<h2>Announcing the release of Keyman 15.0</h2>
+<p class="red">9 May, 2022</p>
+<p>We are pleased to announce that Keyman 15.0 is now available for download!</p>
+<br>
+
+<p><a href="">Read our Keyman 15 launch blog post</a></p>
```

Comment:
```suggestion
```

We've got the beta blog post and this page. Not sure if we also need a release blog post?